app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -73,52 +235,84 @@ class MoveToHousehold extends React.Component {
           console.error(error);
         });
     });
+  };
+
+  /**
+   * Grabs a formatted string with the name of the patient who is being moved to a household.
+   */
+  getPatientName = () => {
+    return `${this.props.patient?.first_name || ''} ${this.props.patient?.middle_name || ''} ${this.props.patient?.last_name || ''}`;
+  };
+
+  /**
+   * Handles a key press event on the search form control.
+   * Checks for enter button press and prevents submisson event.
+   * @param {Object} event
+   */
+  handleKeyPress(event) {
+    if (event.which === 13) {
+      event.preventDefault();
+    }
   }
 
-  createModal(title, toggle, submit) {
+  createModal(title, toggle) {
     return (
-      <Modal size="lg" show centered onHide={toggle}>
+      <Modal dialogClassName="modal-move-household" show centered onHide={toggle}>
         <Modal.Header>
           <Modal.Title>{title}</Modal.Title>
         </Modal.Header>
-        <Modal.Body>
+        <Modal.Body className="modal-move-household-body">
           <Form>
             <Row>
               <Form.Group as={Col}>
-                <Form.Label htmlFor="hoh_selection" className="nav-input-label">
-                  Select The New Monitoree That Will Respond For The Current Monitoree
-                </Form.Label>
-                <Form.Label size="sm" className="nav-input-label">
-                  Note: The current monitoree will be moved into the selected monitoree&apos;s household
+                <Form.Label>
+                  Please select the new monitoree that will respond for <b>{this.getPatientName()}</b>.
                 </Form.Label>
-                <Form.Control as="select" className="form-control-lg" id="hoh_selection" onChange={this.handleChange} defaultValue={-1}>
-                  <option value={-1} disabled>
-                    --
-                  </option>
-                  {this.state?.dependents?.map((member, index) => {
-                    return (
-                      <option key={`option-${index}`} value={member.id}>
-                        {member.last_name}, {member.first_name} Age: {member.age}, State ID: {member.state_id}
-                      </option>
-                    );
-                  })}
-                </Form.Control>
+                <p>
+                  You may select from the provided existing Head of Households and monitorees who are self reporting.
+                  {this.getPatientName()} will be immediately moved into the selected monitoree&apos;s household.
```

Comment:
Gah thanks. This was driving me nuts because in other `p` elements the spaces will show up fine between words when written across lines in the HTML without an `&nbsp;` but in this case that does not appear to be workin, and I want to avoid the `&nbsp;` as it makes the tests frustrating. I'll fix it.

Reply:
Fixed here: 7dac19f